java/BUILD.bazel

```diff
@@ -1,4 +1,4 @@
-load("//bazel:ray.bzl", "flatbuffer_java_library", "define_java_module", "gen_java_pom_file")
+load("//bazel:ray.bzl", "define_java_module", "flatbuffer_java_library", "gen_java_pom_file")
```

Comment:
I don't think the changes in this file are needed. are they unintentional?

Reply:
Oops, sorry - my IDE must have done some style formatting. Undoing now.